lib/sycamore/sycamore/transforms/partition.py

```diff
@@ -413,6 +413,7 @@ def __init__(
         aryn_api_key: str = "",
         aryn_partitioner_address: str = DEFAULT_ARYN_PARTITIONER_ADDRESS,
         use_cache=False,
+        pages_per_call: int = 25,
```

Comment:
How did you pick 25? I'm debating myself whether I think the default should be something like this or just sending the whole doc. 

I do wonder if we can have some value that says send the whole doc, without requiring the caller to know the number of pages in the pdf. Maybe None or -1?

Reply:
Turned off by default.  Added the option `-1` to mean send the whole doc.